config/labels.yml

```diff
@@ -173,11 +173,6 @@ repos:
     <<: *feature_introduced_in_darga_or_prior
     <<: *provider_plugin
     <<: *release
-  manageiq-providers-hawkular:
-    <<: *common
-    <<: *feature_introduced_in_darga_or_prior
-    <<: *provider_plugin
-    <<: *release
```

Comment:
Keep this for now, since we still have the repo in gaprindashvili...we might need to tweak the labels here to use 

```yaml
   <<: *release_darga
   <<: *release_euwe
   <<: *release_fine
   <<: *release_gaprindashvili
```

instead of 

```yaml
    <<: *feature_introduced_in_darga_or_prior
```